lib/gamelogic.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/LineLength
+# rubocop:disable Metrics/CyclomaticComplexity
+class Game
+  attr_accessor :players
```

Comment:
`attr_accessor` is not allowed.
Please, choose between `attr_reader` or `attr_writer`